_posts/2021-06-06-announcing_trino_summit.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "Announcing Trino Summit"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+Greetings Trino Nation,
+
```

Comment:
I don't think "nation" needs to be capitalized, but its not a hill I'm going to die on.

Reply:
haha yup i'll lowercase it